openapi-diff/src/core/OpenApiDiff.Core/Logging/FileObjectPath.cs

```diff
@@ -1,31 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using YamlDotNet.RepresentationModel;
-
-namespace OpenApiDiff.Core.Logging
-{
-    /// <summary>
-    /// Represents a path into an object stored in a file.
-    /// </summary>
-    public class FileObjectPath
```

Comment:
I see the JsonReference and Readable path fields, is there any point in having ObjectPath public ?